### PR TITLE
chore(profile): small style fixes

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**

### CODE REVIEW

File: src/components/tables/base/TableFoot.jsx [hunk 1]
```diff
@@ -20,9 +21,11 @@ class TableFoot extends Component {
         <tr>
           {
             this.props.cols.map((col, i) => (
-              <TCell key={`col_${col.name}_${i}`}>
-                {col.name}
-              </TCell>
+              this.props.colStyles.length === 0 ?
```
Comment (frickjack) [inline src/components/tables/base/TableFoot.jsx]: This check should be `.length > i` ?

File: src/components/tables/base/TableHead.jsx [hunk 2]
```diff
@@ -24,9 +25,15 @@ class TableHead extends Component {
         <tr>
           {
             this.props.cols.map((col, i) => (
-              <ColHead key={`col_${col}_${i}`}>
+              this.props.colStyles.length === 0 ? <ColHead key={`col_${col}_${i}`}>
                 {col}
-              </ColHead>
```
Comment (frickjack) [inline src/components/tables/base/TableHead.jsx]: Check for `.length > i` again ...

File: src/components/tables/base/TableRow.jsx [hunk 3]
```diff
@@ -21,7 +22,9 @@ export const TCell = styled.td`
 
 class TableRow extends Component {
   renderCols = cols => cols.map((col, i) => (
-    <TCell key={`col_${i}`}>{col}</TCell>
+    this.props.colStyles.length === 0 ?
+      <TCell key={`col_${i}`}>{col}</TCell> :
```
Comment (frickjack) [inline src/components/tables/base/TableRow.jsx]: `.length > i` ...

File: src/components/tables/base/TableFoot.jsx (hunk 1 above)
Comment (thanh-nguyen-dang) [reply src/components/tables/base/TableFoot.jsx]: yeah, I'll fix it

File: src/components/tables/base/TableHead.jsx (hunk 2 above)
Comment (thanh-nguyen-dang) [reply src/components/tables/base/TableHead.jsx]: should be <= to fit with current logic

**Pull request merged**